setup.py

```diff
@@ -46,10 +46,8 @@ def get_package_data(name, extlist):
 This is a plugin for the Spyder IDE that integrates popular unit test
 frameworks. It allows you to run tests and view the results.
 
-**Status:**
-This is a work in progress. It is useable, but only the basic functionality
-is implemented at the moment. The plugin currently supports the py.test and nose
-testing frameworks.
+The plugin supports the `unittest` framework in the Python
+standard library and the `py.test` and `nose` testing frameworks.
```

Comment:
FYI, its officially ``pytest`` now

Reply:
Right. I'll use another PR to change this en-masse; see #117.

Reply:
:+1: